schema_source/sam.schema.json

```diff
@@ -186,6 +156,36 @@
       "title": "AppSyncResolver",
       "type": "object"
     },
+    "Authoriser": {
+      "additionalProperties": false,
+      "properties": {
+        "LambdaAuthorizer": {
+          "$ref": "#/definitions/LambdaAuthoriserConfig"
```

Comment:
This doesn't match the GraphQL resource - why are we using "Authoriser" instead of "Authorizer"?